apps/core/scenes/results/Results.js

```diff
@@ -26,12 +30,30 @@ export default class Results extends React.Component<Props> {
   };
 
   render() {
-    const { travelFrom, travelTo, dateFrom, dateTo } = this.props;
+    const {
+      travelFrom,
+      travelTo,
+      dateFrom,
+      dateTo,
+      returnDateFrom,
+      returnDateTo,
+    } = this.props;
+
+    // @TODO get from config
+    const dateFormat = 'Do MMMM';
```

Comment:
Together with the localization of prices, there was also a DateFormatter file: https://github.com/kiwicom/margarita/pull/117/files#diff-cbd47da69efb4e89e3d7340c962d5241

It might be useful here too?

Reply:
Well, is it better in something to use it instead of `date-fns`? Because in this case we would need to take care about logic with ordinal number like (1st, 2nd, 3rd, 4th etc.)